goal_src/jak2/pc/features/speedruns.gc

```diff
@@ -20,6 +19,15 @@
       (initialize! *game-info* 'game (the-as game-save #f) "game-start"))
     (((speedrun-category newgame-heromode))
       (initialize! *game-info* 'game (the-as game-save #f) "game-start-hero"))
+    (((speedrun-category newgame-allopenorbs))
+      (set! (-> *speedrun-menu* 0 draw-menu?) #f)
+      (set-master-mode 'game)
```

Comment:
hiding the menu here shouldn't be required, none of the other categories require that?  The menu automatically hides when hitting "reset".  I believe that should also change the master-mode as well.

Reply:
Ah I understand, kk well as long as it works then maybe it can be cleaned up later.